cmds/tail/tail_test.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"testing"
+)
+
+func TestTailReadBackwards(t *testing.T) {
+	input, err := os.Open("./test_samples/read_backwards.txt")
+	if err != nil {
+		t.Error(err)
+	}
+	output := bytes.NewBuffer([]byte{})
```

Comment:
```go
output := &bytes.Buffer{}
```

is sufficient, bytes.Buffer will allocate bytes as needed.

Reply:
ack